lib/TPP/Dialect/Tpp/TppUtils.cpp

```diff
@@ -272,6 +272,40 @@ bool isTppZero(linalg::GenericOp linalgOp, SmallVectorImpl<Value> *operands) {
   return true;
 }
 
+void splitAndReplaceFusedOp(tpp::FusedBrgemmOp fusedBrgemmOp,
+                            PatternRewriter &rewriter) {
+  assert(fusedBrgemmOp.hasBufferSemantics() &&
```

Comment:
This should be a `return failure();` as we may have this pattern as we move to tensors.